iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/sync/AuthOperationProcedure.java

```diff
@@ -221,19 +227,19 @@ public void deserialize(ByteBuffer byteBuffer) {
     }
     this.timeoutMS = ReadWriteIOUtils.readLong(byteBuffer);
     try {
-      ReadWriteIOUtils.readInt(byteBuffer);
+      int cap = ReadWriteIOUtils.readInt(byteBuffer);
+      int position = byteBuffer.position();
       this.plan = (AuthorPlan) ConfigPhysicalPlan.Factory.create(byteBuffer);
+      byteBuffer.position(cap + position);
     } catch (IOException e) {
       LOGGER.error("IO error when deserialize authplan.", e);
     }
-    if (byteBuffer.hasRemaining()) {
```

Comment:
why removing this judgement? if you think this bytebuffer will always has remaining, you should add a else branch and then throw exeception